### ISSUE TITLE
Test develop branch

### ISSUE THREAD

**Issue by jonathanbossenger** [hlashbrooke/WordPress-Plugin-Template#90]
Calling all active participants/users of this plugin template.

I've recently merged all PRs that could be merged into the develop branch. Before merging these changes into master, I'd like to have some folks test the plugin template out, to make sure it still works as expected, as well as with the new changes.

If one of your PR's was merged, please do test out your specific change.

Once I have a some feedback on this ticket, I will be able to make a call to merge the changes.

Thank you

**Comment by jonathanbossenger** [comment]
pinging

@e-baker 
@alekseyn1
@carl-alberto 
@cornelRaiu 

Also @hlashbrooke if you have some time :-)

**Comment by e-baker** [comment]
Looks good for me!

**Comment by carl-alberto** [comment]
Looking good to me so far

**Comment by jonathanbossenger** [comment]
Thanks for the feedback everyone. I will look at merging these changes into the master branch shortly.

In another note, I want to try and see if we can put together a little community around this plugin template, and start looking at how we could get it to the same level of usage/interest as the WP Plugin Boiler Plate, but using more modern methods, similar to how WP Rig has done for themes (namespaces, autoloading, block ready etc etc).

Would you all be interested in starting a Slack instance to get this moving?

**Comment by alhoseany** [comment]
This is a great idea. count me in.
thanks

**Comment by alhoseany** [comment]
I have created multiple fields for the plugin like, Query table, file upload, image search and even ajax buttons. Once I have some free time I will create a PR.
Screenshots:
https://prnt.sc/s19lg4
https://prnt.sc/s19nea

**Comment by hlashbrooke** [comment]
This is all working for me - nice job everyone!

I think a community around this is a great idea :) happy to assist with that in any way I can.

**Comment by cornelRaiu** [comment]
Hi everyone!
Yes, this is an awesome idea!
I am going to use this template in some of the plugins I want to develop and launch in the WP repo this year and rewrite the one I already launched.

Furthermore, I will build all the plugins on live coding streams so that will be some nice exposure to the template. 

It would be nice to see this one growing.

**Comment by carl-alberto** [comment]
Yeah, happy to have a community around this project! Let's do this! How do we start @jonathanbossenger ? :)

**Comment by jonathanbossenger** [comment]
Glad to hear everyone is excited about this. For now, just join the slack team I just created here

https://join.slack.com/t/wp-plugin-workspace/shared_invite/zt-dlbdpxk6-WVeY_M7K16nMF5XosTC9rA

I'll start a document next week with my ideas for the project, and we can go from there.

**Comment by jonathanbossenger** [comment]
@alantygel and anyone else, I've just merged one more PR into develop, please give everything one more test, if it's all fine, I can merge the changes in master. Thanks

**Comment by ricklambrechts** [comment]
Hi,

Today i was starting a new wordpress plugin project and was using the development branch for testing.

The whole time i was tinking that the build-plugin.sh script was not working using Git Bash and other terminals i tried.  It was showing me this error `bash: [: ==: unary operator expected`. But then after a few hours, i realised. Wait.. maybe i shoud type the Class name, because the file names renamed are missing the class name.
So after typing the class name, of course.. no problem.

Maybe the README.md needs an update now because of the added Class name.
